numpyro/svi.py

```diff
@@ -147,7 +132,7 @@ def evaluate(rng, opt_state, model_args=(), guide_args=()):
     return init_fn, update_fn, evaluate
 
 
-def elbo(param_map, model, guide, model_args, guide_args, kwargs, constrain_fn, is_autoguide=False):
+def elbo(param_map, model, guide, model_args, guide_args, kwargs):
```

Comment:
@neerajprad I removed `constrain_fn` for clarity. We no longer expose `constrain_fn` for users so we'd better remove it. In addition, I removed `is_autoguide` arg and move the check `isinstance(guide, AutoContinuous)` from svi to inside of elbo to clean up the interface. We'll address issues of autoguide in another PR. I think we should set default values to args, kwargs here but let's defer it to another PR.